packages/services/f-feature-management/src/index.js

```diff
@@ -0,0 +1,34 @@
+function logValueRequest (func, key, value) {
```

Comment:
We generally prefer the ES6 syntax:
```suggestion
const logValueRequest = (func, key, value) => {
```